src/lib/utils/number.ts

```diff
@@ -0,0 +1,7 @@
+export function isNumber(input: unknown): input is number {
```

Comment:
This is wrong, since this would make a numeric `string` appear as `number` in TS.